capreolus/utils/trec.py

```diff
@@ -96,3 +97,39 @@ def load_qrels(qrelfile, qids=None, include_spam=True):
 
     labels.default_factory = None  # behave like normal dict
     return labels
+
+
+def document_to_trectxt(docno, txt):
```

Comment:
maybe I also move the `document_file_to_trectxt_file` ([like this](https://github.com/crystina-z/capreolus-1/blob/master/capreolus/collection/__init__.py#L76-L86)) to here?

Reply:
yep, you could call `document_to_trectxt` instead of formatting it inside antique